its/projects/ReproAzureFunctions/ReproAzureFunctions/ReproAzureFunctions.csproj

```diff
@@ -7,9 +7,9 @@
     <Nullable>enable</Nullable>    
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Http" Version="3.0.12" />
-    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.3.0" OutputItemType="Analyzer" />
-    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.6.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Http" Version="3.2.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.17.2" OutputItemType="Analyzer" />
```

Comment:
Looks good. The recommendation is to "Upgrade to version SharpZipLib - 1.3.3"
![image](https://github.com/SonarSource/sonar-scanner-msbuild/assets/103252490/6cd3a64c-7ece-418b-9e70-4d95b11063b2)